docarray/base_doc/doc.py

```diff
@@ -61,39 +61,12 @@
 ExcludeType = Optional[Union['AbstractSetIntStr', 'MappingIntStrAny']]
 
 
-class BaseDoc(BaseModel, IOMixin, UpdateMixin, BaseNode):
+class BaseDocWithoutId(BaseModel, IOMixin, UpdateMixin, BaseNode):
     """
-    BaseDoc is the base class for all Documents. This class should be subclassed
-    to create new Document types with a specific schema.
-
-    The schema of a Document is defined by the fields of the class.
-
-    Example:
-    ```python
-    from docarray import BaseDoc
-    from docarray.typing import NdArray, ImageUrl
-    import numpy as np
-
-
-    class MyDoc(BaseDoc):
-        embedding: NdArray[512]
-        image: ImageUrl
-
-
-    doc = MyDoc(embedding=np.zeros(512), image='https://example.com/image.jpg')
-    ```
-
-
-    BaseDoc is a subclass of [pydantic.BaseModel](
-    https://docs.pydantic.dev/usage/models/) and can be used in a similar way.
+    BaseDocWoId is the class behind BaseDoc, it should not be used directly unless you know what you are doing.
```

Comment:
Add that it will not be possible to use if with DocumentIndices

Reply:
good point

Reply:
done